raiden/transfermanager.py

```diff
@@ -86,68 +128,172 @@ def transfer_async(self, amount, target, identifier=None, callback=None):
         """
 
         # Create a default identifier value
-        if not identifier:
+        if identifier is None:
```

Comment:
I am okay with either as long as we decide on a consisten check for the identifier.

I am not sure how is the "pythonic" way to handle such checks. Indeed identifier should be either an integer or `None`. Any other values here would be a mistake. The question is should these mistaken values be allowed to be assigned to identifier? Say an empty list?

I have no idea how such cases should be handled so I am curious to know.